internal/slicer/report.go

```diff
@@ -30,37 +32,38 @@ type Report struct {
 
 // NewReport returns an empty report for content that will be based at the
 // provided root path.
-func NewReport(root string) *Report {
-	return &Report{
+func NewReport(root string) (*Report, error) {
+	if !filepath.IsAbs(root) {
+		return nil, fmt.Errorf("cannot use relative path for report root: %q", root)
```

Comment:
This was discussed in a different PR, so there are probably mixed diffs here. Can we please get this PR updated so it reflects the current delta against main?  Thanks.